code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -93,6 +93,8 @@
 
 /datum/reagent/medicine/tramadol/on_mob_life(mob/living/L)
 	L.reagent_pain_modifier += PAIN_REDUCTION_VERY_HEAVY
+	if(volume > 5)
+		L.reagent_move_delay_modifier = min(0.5, volume * 0.013)
```

Comment:
Don't use `reagent_move_delay_modifier `, we want to eliminate this.
Use `add_movespeed_modifier` and `remove_movespeed_modifier` on `on_mob_add` and `on_mob_delete`

Reply:
Nothing else has that, and I didn't want to add a competing standard, like the 3 separate purgechem definitions.

I'd like to get this TM'd to see how it plays before I go through and change all of them.

Reply:
It's not a competing standard as much as the standard we are moving towards.

Do this properly and it will get TMed and merged eventually. Else you are leaving someone else to clean it up.

Reply:
Fine, fine, I'll take a look and muddle through changing all of them.

What standard do you want for purgecode?

Reply:
In fact, this may be due to limited coding experience, but I'm not sure how to add a gradual movespeed modifier based off of volume on that, and I really don't want to have a fixed modifier assigned to every chem.

I'll see if I can find some examples in regular TGcode.

Reply:
All reagant movespeed modifiers removed, replaced with minor to moderate pain.